src/app/dashboard/offerings/components/OfferingItem.jsx

```diff
@@ -0,0 +1,88 @@
+import { Accordion, AccordionContent, AccordionTitle, AccordionPanel, Badge } from 'flowbite-react'
+import { HiIdentification, HiCalendar, HiLink, HiOutlineScale } from 'react-icons/hi'
+import OfferingActions from './OfferingActions'
+
+export default async function OfferingItem ({ offering }) {
+  // For now only checking & defaulting keywords, but others could be optional fields...
+  // Base offering details
+  const offerId = offering['@id']
+  const title = offering['dct:title']?.['@value']
+  const license = offering['dct:license']?.['@value']
+  const issueDate = offering['dct:issued']?.['@value']
+
+  // Handle first asset from 'sedi:hasAsset' array (if available)
+  const firstAsset = Array.isArray(offering['sedi:hasAsset'])
+    ? offering['sedi:hasAsset'][0]
+    : null
+
+  const keywords = firstAsset?.['dcat:keyword']
+    ? firstAsset['dcat:keyword'].map(k => k['@value'])
+    : []
+
+  const assetUrl = firstAsset?.['sedi:isProvidedBy']?.['dcat:accessURL']?.['@id']
+  const description = firstAsset?.['sedi:isProvidedBy']?.['dct:description']?.['@value']
+
+  return (
+    <Accordion collapseAll className='min-w-fit overflow-auto mx-4 mb-4 shadow-md rounded-md'>
+      <AccordionPanel>
+        <AccordionTitle className='bg-white'>
+          <div className='grid grid-cols-2'>
+            <div>
+              <div className='flex text-lg font-semibold'>
+                <div className='flex flex-row'>
+                  <p>{title}</p>
+                </div>
+              </div>
+              <div className='flex flex-col mt-4 min-w-40'>
+                <div className='text-sm mr-4'>
+                  <div className='flex flex-row'>
+                    <HiIdentification size={20} className='mr-1' />
+                    <p>{offerId}</p>
+                  </div>
+                </div>
+                <div className='flex flex-row items-end mt-3'>
+                  <div className='flex flex-row gap-2 w-36 mt-2'>
```

Comment:
Important: we need to remove the width here, otherwise the date field and the link overlaps. Moreover, we need to add a `gap` to adjust the spacing between the icon and the value. For the latter, you can alternatively set a `className="mr-1"` in the icon component, to be consistent with what you do with the offering id. Up to you.
```suggestion
                  <div className='flex flex-row gap-2 mt-2 gap-1'>
```